packages/fxa-content-server/app/styles/modules/_button-row.scss

```diff
@@ -69,39 +68,6 @@ button {
     }
   }
 
-  &.warning-button {
```

Comment:
We don't have anything using this or `settings-button` in the codebase so I removed those styles.